services/algolia-sync/src/components/algolia/message-sync.js

```diff
@@ -37,6 +38,11 @@ const upsertToIndex = async (message) => {
 
     c.boost = boostResult(c);
 
+    // Set unpublished date to 100 years in the future if it's null
+    if (c.unpublished) {
```

Comment:
This should be `if (!c.unpublished)` note the **!** not qualifier

Reply:
Oops, I fixed that quick!